notebooks/2023-01-24-dhs-cross-country-experiments/2023-02-06_crosscountry_normalized.ipynb

```diff
@@ -0,0 +1,1476 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.    print(f'Mean split r^2  (std): {split_r2_mean} ({split_r2_std})')
</pre><p>Last thing, can we please update the code here to reflect the best results/recommendations you have? </p><p> </p><p>You mentioned MinMaxScaler for features, then StandardScaler for the Wealth Index. These results and scatterplots seem to be different from the ones you previously reported. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Notebook scaling function and cells are now fixed, and now displays best config: per-country `StandardScaler` for both feature and label columns.